app/Trader.php

```diff
@@ -64,6 +66,25 @@ public function users()
         return $this->belongsToMany(User::class);
     }
 
+    /**
```

Comment:
Not sure i actually use these anywhere...

Reply:
DO you want to check and delete them? Or use them?

Reply:
I think they'll be useful for tinkering.